sys-libs/libxcrypt/libxcrypt-4.4.28-r2.ebuild

```diff
@@ -238,18 +247,16 @@ src_install() {
 	) || die "failglob error"
 
 	# Remove useless stuff from installation
-	find "${ED}"/usr/share/doc/${PF} -type l -delete || die
+	find "${ED}"/usr/share/doc/${PF} -delete || die
```

Comment:
I don't think we want to unconditionally kill these, even though I accept it's probably not useful for most people.